jax_distribution/systems/ff_ippo_rware.py

```diff
@@ -385,10 +389,14 @@ def run_experiment(_run: run.Run, _config: Dict, _log: SacredLogger) -> None:
     # Create envs
     env = jumanji.make(config["env_name"], generator=generator)
     env = RwareMultiAgentWrapper(env)
+    if config["add_agent_id"]:
```

Comment:
you can make both env and eval_env under one if condition as the remaining code after env is independent of it

Reply:
The reason for having two if statements is because we need the training environment to be wrapped in the `AutoReset` and `Log` wrappers. So doing both at the end would cause trouble as far as I can see.

Reply:
We can write it this way: 
```
env = jumanji.make(config["env_name"], generator=generator)
env = RwareMultiAgentWrapper(env)
eval_env = RwareMultiAgentWrapper(env)
if config["add_agent_id"]:
        env = AgentIDWrapper(env)
        eval_env = AgentIDWrapper(eval_env)
env = AutoResetWrapper(env)
env = LogWrapper(env)
```